dashboard/app/models/ability.rb

```diff
@@ -166,6 +166,17 @@ def initialize(user)
       end
     end
 
+    if user.permission?(UserPermission::LEVELBUILDER)
+      can :manage, [
+        FrequentUnsuccessfulLevelSource,
+        Level,
+        LevelSourceHint,
+        Script,
+        ScriptLevel,
+        Stage,
+      ]
+    end
```

Comment:
do we need to add the same `cannot [:update, :destroy], Level do |level| !level.custom? end` addendum as we do for admins?